SplitFiles/game.c

```diff
@@ -64,14 +63,19 @@ int wait_for_a_key(GameComponent* input, GameParameters* params) {
 }
 
 void tick_func(GameComponent* input, GameParameters* params) { //Now a scoreboard too
-  int last_score = (int)input->ptr;
+  long last_score = (long)input->ptr;
   if (params->score != last_score) {
-    screenNumber(21, 1, params->score);
+    int show_score = (int)(params->score >> 3);
+    screenNumber(21, 1, show_score);
   //  printf(PRINTAT"\x19\x01""%6d", params->score);
```

Comment:
I think the issue here is that we have a score like this 0x60000 and shift that to the left by three we get 0xE000 which is still negative. I think I saw a suggestion from you before that a negative score should mean you've lost and that's the way round this and make these things unsigned?

Reply:
The other thing, of course, is to stop the score escalating so quickly!